jobtweets.py

```diff
@@ -50,16 +50,26 @@ def get_tweet_sentiment(self, tweet):
         else:
             return 'negative'
  
-    def get_tweets(self, query, count = 10):
+    def get_tweets(self, count = 10):
         '''
         Main function to fetch tweets and parse them.
         '''
-       
+        queries = ""
+        topicList = []

```

Comment:
```suggestion
        topic_list = []
```
Nit: PEP8 has a section that talks about [function and variable names](https://www.python.org/dev/peps/pep-0008/#function-and-variable-names). It is advised, and you will see this more often than not, that variable and function names be written in snake case unless mixedCase is the prevailing style. I took a look at the files and it looks like the prevailing style is snake_case.